src/containers/assignees/reducer.js

```diff
@@ -0,0 +1,61 @@
+import {
+  ADD_ASSIGNEE_TO_CARD, ADD_RESPONSIBLE_TO_CARD, GET_ALL_ASSIGNEES_IN_CARD, GET_RESPONSIBLE_FOR_CARD,
+  REMOVE_ASSIGNEE_FROM_CARD, REMOVE_RESPONSIBLE_FROM_CARD } from './constants'
+
+const initialState = {
+  assignees: [],
+  responsible: {},
```

Comment:
Don't store the responsible here, it's a raw value directly in the CardContainer